src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -25,13 +31,24 @@ interface PharmacyProps {
 export function PharmacyCard(props: PharmacyProps) {
   const [booking, setBooking] = useState(props.booking);
   const [lastUpdated, setlastUpdated] = useState(props.lastUpdated);
+  const [shouldShowMap, setShouldShowMap] = useState(false);
 
   const updatePharmacy = (availability: boolean) => {
     const time = new Date();
     setlastUpdated(time.toLocaleString());
     setBooking(availability);
   };
 
+  const Map = () => (
+    <Iframe
```

Comment:
Can we add styling to either explicitly define the border, or (my preferred option) remove the border on this iframe